tools/nni_trial_tool/trial_keeper.py

```diff
@@ -99,6 +98,7 @@ def trial_keeper_help_info(*args):
     PARSER.add_argument('--pai_hdfs_host', type=str, help='the host of hdfs')
     PARSER.add_argument('--pai_user_name', type=str, help='the username of hdfs')
     PARSER.add_argument('--nni_hdfs_exp_dir', type=str, help='nni experiment directory in hdfs')
+    PARSER.add_argument('--disable_log', type=bool, default=False, help='disable trial log')
```

Comment:
This causes confusion,  --disable_log False  or --disable 'False'   are evaluated as True.  
Typical solution for bool argument are :  PARSER.add_argument('--disable_log', action='store_true'), so that you do not need to pass 'True' or 'False'